### PR TITLE
Stick to your own format "s <query> [flags]"

### PR DESCRIPTION
**Pull request by KeizerDev**
When looked into the project again I founded you're not really sticking to your own format. :wink:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,24 +32,24 @@ s puppies
 
 Search for a wifi router on amazon
 ```
-s -p amazon wifi router
+s wifi router -p amazon
```
Comment (KeizerDev) [inline README.md]: You should actually change `wifi router` to `"wifi router"` just to make sure there won't be any issues with using a dash.

**Comment by zquestz** [comment]
I actually prefer the flags first so that the phrase is more clear.

**Comment by zquestz** [comment]
Its probably from the hack I did in cobra, as it doesn't intend for a full search query. The help is just slightly off because we either get "s [flags]" or "s <query> [flags]" but there is no nice way to instrument "s [flags] <query>" using the library.

**Comment by KeizerDev** [comment]
That's true. So I'll just close my pr.

**Pull request closed without merging**